### PR TITLE
[release/v0.5.0] History - Correction of problems with the logger

### PR DESCRIPTION
**Pull request by thiagodpf**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

bug fix

* **What is the current behavior?** (You can also link to an open issue here)

- There were not many logs in the component;
- When the LOG_LEVEL environment variable was set to a minuscule value, there was a problem with the component initialization;
- When the environment variable LOG_LEVEL was initialized with the value INFO and later, if the admin wanted to change this value to DEBUG, the debug messages were not logged.

* **What is the new behavior (if this is a feature change)?**

- more logs have been added to the functions;
- env_var LOG_LEVEL value converted to uppercase;
- The error that occurred when trying to change the log level has been fixed (the level of the StreamHandler of the log was not being changed).

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

no

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

dojot/dojot#1513
dojot/dojot#1515

* **Other information**:

I changed the server port in standalone mode to 8787 because port 8000 is commonly used by Kong in the dojot docker-compose, so to be able to run the component (in standalone mode for debug) together with the dojot, this change was necessary.

### CODE REVIEW

File: history/app.py [hunk 1]
```diff
@@ -16,11 +16,10 @@
...
 app.add_route('/notifications/history', NotificationHistory())
 app.add_route('/STH/v1/contextEntities/type/{device_type}/id/{device_id}/attributes/{attr}', STHHistory())
-app.add_route('/log',LoggingInterface())
+app.add_route('/log', LoggingInterface())
 
 # Useful for debugging problems in API, it works with pdb
 if __name__ == '__main__':
     from wsgiref import simple_server  # NOQA
-    httpd = simple_server.make_server('127.0.0.1', 8000, app)
+    httpd = simple_server.make_server('127.0.0.1', 8787, app)
```
Comment (mprevide) [inline history/app.py:24]: Be careful about change this port. Please, revert to 8000.

File: history/conf.py [hunk 2]
```diff
@@ -8,8 +8,8 @@
 # This file contains the default configuration values
 # and configuration retrieval functions
 
-#Logger related configuration
-log_level = os.environ.get("LOG_LEVEL","INFO")
+# Logger related configuration
+log_level = os.environ.get("LOG_LEVEL", "INFO").upper()
```
Comment (mprevide) [inline history/conf.py:12]: Good solution :smile:

**Comment by mprevide** [comment]
I like your changes but the codecov maybe want a little more :smiley:

File: history/app.py (hunk 1 above)
Comment (thiagodpf) [reply history/app.py:24]: You are right, I thought that this section was intended for standalone development only, I will return to the current value.

Comment (mprevide) [reply history/app.py:24]: I think you all right, but I think It's better not change that.

**Review by mprevide — APPROVED** [review]

**Pull request merged**